react-components/package.json

```diff
@@ -20,9 +20,9 @@
         "prop-types": "^15.6.2"
     },
     "dependencies": {
-        "@cyverse-de/ui-lib": "0.3.4",
-        "@material-ui/core": "^3.9.3",
-        "@material-ui/icons": "^3.0.2",
+        "@cyverse-de/ui-lib": "file:../../ui-lib",
```

Comment:
This should probably be the version pushed instead of a path

Reply:
Correct. I have changes to merge in https://github.com/cyverse-de/ui-lib/pull/12. Once that is approved and merged, I will update this.